cmd/mesh_bridge/main.go

```diff
@@ -0,0 +1,106 @@
+// +build !js
+
+// mesh_bridge is a short program that bridges two Mesh nodes. This is useful in cases where
+// we introduce a network-level breaking change but still want the liquidity from one network
+//to flow to another
+package main
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/0xProject/0x-mesh/rpc"
+	"github.com/0xProject/0x-mesh/zeroex"
+	"github.com/plaid/go-envvar/envvar"
+	log "github.com/sirupsen/logrus"
+)
+
+type clientEnvVars struct {
+	FirstWSRPCAddress  string `envvar:"FIRST_WS_RPC_ADDRESS"`
+	SecondWSRPCAddress string `envvar:"SECOND_WS_RPC_ADDRESS"`
+	Verbose            bool   `envvar:"VERBOSE"`
```

Comment:
Why not let this be an `int` and call it `Verbosity` like we do for our other executables? It gives users more control and doesn't require any extra lines of code to support.